devtools-ci-index.yaml

```diff
@@ -2218,22 +2218,6 @@
             saas_service_name: rh-che6
             timeout: '30m'
             extra_target: rhel
-        - '{ci_project}-{git_repo}-build-che-credentials-{branch}':
-            git_organization: redhat-developer
-            git_repo: rh-che
-            ci_project: 'devtools'
-            branch: master
-            ci_cmd: '/bin/bash .ci/cico_build.sh && /bin/bash .ci/cico_deploy.sh'
-            saas_git: saas-openshiftio
-            timeout: '20m'
-        - '{ci_project}-{git_repo}-build-che-credentials-{branch}':
-            git_organization: redhat-developer
-            git_repo: rh-che
-            ci_project: 'devtools'
-            branch: single-user-rh-che
```

Comment:
@riuvshin please, do not remove `single-user-rh-che` branch 
@l0rd am I correct that we have decided to leave it for the case when we might need single-tenant che in oso ?

Reply:
@ibuziuk I don't remember that. I think we kept that because in case we wanted to rollback multi-user deployment but I may be wrong. In what cases do you think we might need single tenant?

Reply:
@l0rd do we still need `single-user-rh-che` ?

Reply:
@riuvshin no we don't need it anymore